.github/workflows/stale.yml

```diff
@@ -9,9 +9,9 @@ jobs:
       - uses: actions/stale@v4
         with:
           stale-issue-message: 'This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.'
```

Comment:
```suggestion
          stale-issue-message: 'This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contribution!'
```